kalite/coachreports/views.py

```diff
@@ -154,8 +154,9 @@ def tabular_view(request, report_type="exercise"):
     # Define how students are ordered--used to be as efficient as possible.
     student_ordering = ["last_name", "first_name", "username"]
 
-    # Get a list of topics (sorted) and groups
-    topics = [get_node_cache("Topic", language=language).get(tid["id"]) for tid in get_knowledgemap_topics(language=language) if report_type.title() in tid["contains"]]
+    # Get a list of topics that are at the third level down of the topic tree to use as items in the drop down.
```

Comment:
Not really. Need to come up with a better way, but I am just replicating
our historical functionality here.

On Wed, 13 May 2015 17:02 Michael Gallaspy <notifications@github.com> wrote:

> In kalite/coachreports/views.py
> <https://github.com/learningequality/ka-lite/pull/3636#discussion_r30287984>
> :
>
> > @@ -154,8 +154,9 @@ def tabular_view(request, report_type="exercise"):
> >      # Define how students are ordered--used to be as efficient as possible.
> >      student_ordering = ["last_name", "first_name", "username"]
> >
> > -    # Get a list of topics (sorted) and groups
> > -    topics = [get_node_cache("Topic", language=language).get(tid["id"]) for tid in get_knowledgemap_topics(language=language) if report_type.title() in tid["contains"]]
> > +    # Get a list of topics that are at the third level down of the topic tree to use as items in the drop down.
>
> Is the third level special?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3636/files#r30287984>.
>